staging/src/k8s.io/csi-translation-lib/plugins/gce_pd.go

```diff
@@ -286,7 +291,10 @@ func (g *gcePersistentDiskCSITranslator) TranslateCSIPVToInTree(pv *v1.Persisten
 		gceSource.Partition = int32(partInt)
 	}
 
-	// TODO: Take the zone/regional information and stick it into the label.
+	// translate CSI topology to In-tree topology for rollback compatibility
+	if err := translateTopologyFromCSIToInTree(pv, GCEPDTopologyKey); err != nil {
```

Comment:
we may want to add the labels too in case people have tooling on the kubernetes labels.

Reply:
The labels of migrated PV already exist in the current case even before it goes to label admission controller.

Reply:
The PV.Labels are set?

Reply:
Oops, it is not... Sorry I had a misunderstanding. Will add it.

Reply:
I add the zone labels in if it does not exist. But I dont know if we can derive region info from Zone values for all the plugins. So I did not add the region label.

An alternative solution is to add a `generateRegionFromZone` func as a parameter to let each plugin pass a region parser.

Reply:
We only have 5 plugins to support. I think it's not that critical to have a common util function and each plugin can parse region their own ways.